crochet-macros/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "crochet-macros"
+version = "0.1.0"
+authors = ["Maan2003 <manmeetmann2003@gmail.com>"]
+edition = "2018"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+syn = { version = "1.0.41", features = ["full"] }
```

Comment:
Is "full" required here? Perhaps a more targeted list could reduce compile time.

Reply:
[ItemFn](https://docs.rs/syn/1.0.42/syn/struct.ItemFn.html)  is only available with full.